src/backend/core/tests/teams/test_core_api_teams_update.py

```diff
@@ -120,8 +120,12 @@ def test_api_teams_update_authenticated_administrators():
     team.refresh_from_db()
     final_values = serializers.TeamSerializer(instance=team).data
     for key, value in final_values.items():
-        if key in ["id", "accesses"]:  # pylint: disable=R1733
+        if key in ["id", "accesses", "created_at"]:  # pylint: disable=R1733
             assert value == initial_values[key]
+        elif key in ["updated_at"]:  # pylint: disable=R1733
+            assert value is not None
+            assert initial_values[key] is not None
```

Comment:
Why are you checking these values against None ?

Reply:
We don't want the values to be None.

Reply:
Your test is not that great. In fact, we want to test that the `updated_at` new value is superior to the initial value. If these fields were not serialized, the key won't be in the final values, I am wrong? Testing for None if the Key is in the serialized values doesn't add value. I red quickly the test I might be wrong;

Reply:
[74cc19b](https://github.com/numerique-gouv/people/pull/55/commits/74cc19b657d702d9a4bddc24b8c224c477667c3a)

> If these fields were not serialized, the key won't be in the final values, I am wrong? Testing for None if the Key is in the serialized values doesn't add value. 

I don't understand, were am I checking the keys are at None? 
I am checking the **value** of the key (`initial_values[key]`), so yes the **value** of the keys could be at None for any unexpected reason (upgrade / change library / modify factory), and we don't want that, it will assert that there is a problem.

Checking things are at None or not are common usecase:
https://github.com/numerique-gouv/people/blob/c6731783cbc1880d16ce708e6638bf3aeb621d42/src/backend/core/tests/test_api_team_accesses.py#L51-L53

Reply:
Let's try to keep it good spirit, this kind of comments are just annoying:

> Your test is not that great. In fact, we want to test that the updated_at new value is superior to the initial value.

Reply:
That why I have edited my comment

Reply:
Please, I explicitly stated `I read quickly the test I might be wrong;` I still maintain that we should test that `value ` is superior to `initial_values[key]`.

Reply:
A fixup has been push [74cc19b](https://github.com/numerique-gouv/people/pull/55/commits/74cc19b657d702d9a4bddc24b8c224c477667c3a), you can see it in this conversation as well: https://github.com/numerique-gouv/people/pull/55#discussion_r1500449260